tgui/packages/tgui/interfaces/ReagentGrinder.js

```diff
@@ -100,7 +109,152 @@ const GrinderContents = (props, context) => {
   );
 };
 
-const GrinderReagents = (props, context) => {
+const TankReagents = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    beaker_loaded,
+    beaker_max_volume,
+    tank_current_volume,
+    tank_max_volume,
+    tank_contents,
+  } = data;
+
+  let [dispenseAmount, setDispenseAmount] = useSharedState(
+    context,
+    'dispenseAmount',
+    1
+  );
+
+  return (
+    <Section
+      title="Tank"
+      fill
+      scrollable
+      height="80%"
+      buttons={
+        <Box>
+          <Box inline color="label" mr={2}>
+            {tank_current_volume} / {tank_max_volume} units
+          </Box>
+          <Button
+            content="Clear Tank"
+            color="red"
+            tooltip="completely empty the internal tank"
+            onClick={() => act('clear')}
+          />
+          <Button
+            content="Dispense Mix"
+            tooltip="Dispense a proportional mix of the tank's contents"
+            onClick={() => act('dispense_mix', { amount: dispenseAmount })}
+          />
+          <NumberInput
+            width="40px"
+            minValue={0}
+            value={dispenseAmount}
+            maxValue={beaker_loaded ? beaker_max_volume : 0}
+            step={1}
+            stepPixelSize={3}
+            onChange={(e, value) => setDispenseAmount(value)}
+          />
+        </Box>
+      }
+    >
+      <Stack fill vertical>
+        <Section fill scrollable title={'Reagents'}>
+          {!tank_contents && (
```

Comment:
```suggestion
          {!tank_contents && (
```
Use ternary 
{tank contents ? (Botanitank content) : (The Botanitank is empty content)}

Reply:
done